### PR TITLE
Yc/zigbee dev

### PR DESCRIPTION
**Pull request by yoelrc88**
Adding new protos for new zigbee driver
Removing old zigbee protos

### CODE REVIEW

File: malos/driver.proto [hunk 1]
```diff
@@ -39,6 +39,8 @@ message DriverConfig {
 
   // Microphone Array configuration
   MicArrayParams  micarray = 10;  
+
```
Comment (arhuaco) [inline malos/driver.proto]: Extra spaces?

File: malos/driver.proto [hunk 2]
```diff
@@ -233,73 +235,332 @@ message EverloopImage {
...
-    TOGGLE = 2;
-    LEVEL = 3;
-    IDENTIFY = 4;
-    COLOR = 5;
   }
 
-  // Bulb commands
-  EnumCommands command = 2;
+  // ZigBee Light Link command definition
+  message ZLLCmd{}
```
Comment (arhuaco) [inline malos/driver.proto]: Empty message?

File: malos/driver.proto [hunk 3]
```diff
@@ -233,73 +235,332 @@ message EverloopImage {
...
+
+    // Color Control command definition
+    message ColorControlCmd {
+
+      // ZCL ColorControl command types
+      enum ZCLColorControlCmdType  {
+        MOVETOHUE = 0;
+        MOVETOSAT = 1;
+        MOVETOHUEANDSAT = 2;
+        // MOVEHUE = 3;
```
Comment (arhuaco) [inline malos/driver.proto]: Just add a note here. Something like:

// We do not yet support these commands

or

// We have no plans support these commands

or ...

File: malos/driver.proto (hunk 1 above)
Comment (yoelrc88) [inline malos/driver.proto]: ok

File: malos/driver.proto (hunk 3 above)
Comment (yoelrc88) [inline malos/driver.proto]: ok, I removed them

File: malos/driver.proto (hunk 2 above)
Comment (yoelrc88) [inline malos/driver.proto]: removed

**Pull request merged**